examples/player/pipeline_bt_sink/main/play_bt_music_example.c

```diff
@@ -118,8 +133,11 @@ void app_main(void)
                      music_info.sample_rates, music_info.bits, music_info.channels);
 
             audio_element_setinfo(i2s_stream_writer, &music_info);
-            i2s_stream_set_clk(i2s_stream_writer, music_info.sample_rates, music_info.bits, music_info.channels);
-            continue;
+#if (CONFIG_ESP_LYRATD_MSC_V2_1_BOARD || CONFIG_ESP_LYRATD_MSC_V2_2_BOARD)
+#else         
+		 i2s_stream_set_clk(i2s_stream_writer, music_info.sample_rates, music_info.bits, music_info.channels);
```

Comment:
i2s_stream_set_clk might be the clincher. Will give it a try.

Reply:
Thanks, i'm a big fan of your chess engine!